### PR TITLE
doc: Minor updates to Bluetooth samples

### PR DESCRIPTION
**Pull request by divipillai**
Made following changes to some of the Bluetooth
samples:
* Mentioned Thingy:53 also as tabs in the user-interface section.
* Moved a note to shortcuts.txt, which was mentioned in different Bluetooth samples.
* Removed trademark symbol apart from the 1st instance.

### CODE REVIEW

**Review by kapi-no — APPROVED** [review]
Thingy:53 UI reorganization looks good (thanks for aligning it with our Fast Pair samples). It would still be good to get a review from the maintainers of this sample set

File: doc/nrf/shortcuts.txt
Comment (MarekPieta) [inline doc/nrf/shortcuts.txt:232]: Consider mentioning nRF Desktop in link name - e,g. `nrf_desktop_HID_ref` (this actually works as a reference to nRF Desktop application)

**Review by MarekPieta — APPROVED** [review]

**Review by KAGA164 — APPROVED** [review]

**Pull request merged**
